File: python/examples/os/fork_skeleton.py
files = glob.glob("*.dna")
count = len(files)
batch = count/n

#files[0 : batch]
#files[batch : 2 * batch]
#...
#files[(n-1) * batch : n * batch]

for i in range(0, n):
   pid = fork()
   if pid:
       # parent
   else:
       # child
       child(files[i * batch : (i+1) * batch])

File: perl/examples/forks/fork_skeleton.pl
use strict;
use warnings;

main();

sub main {
    my $pid = fork();
    die "Could not fork" if not defined $pid;

    if (not $pid) {
        print "In child process\n";
        exit;
    }

    print "In parent process\n";
    my $finished = wait();
}

